conf/messages.en

```diff
@@ -210,6 +210,202 @@ internationalAddress.error.line1.length = Address line 1 must be 100 characters
+country.GN = Guinea
+country.GW = Guinea-Bissau
+country.GY = Guyana
+country.HT = Haiti
+country.HN = Honduras
+country.HU = Hungary
+country.IS = Iceland
+country.IN = India
+country.ID = Indonesia
+country.IR = Iran
+country.IQ = Iraq
+country.IE = Ireland
+country.IL = Israel
+country.IT = Italy
+country.CI = Ivory Coast
+country.JM = Jamaica
+country.JP = Japan
+country.JO = Jordan
+country.KZ = Kazakhstan
+country.KE = Kenya
+country.KI = Kiribati
+country.XK = Kosovo
+country.KW = Kuwait
+country.KG = Kyrgyzstan
+country.LA = Laos
+country.LV = Latvia
+country.LB = Lebanon
+country.LS = Lesotho
+country.LR = Liberia
+country.LY = Libya
+country.LI = Liechtenstein
+country.LT = Lithuania
+country.LU = Luxembourg
+country.MG = Madagascar
+country.MW = Malawi
+country.MY = Malaysia
+country.MV = Maldives
+country.ML = Mali
+country.MT = Malta
+country.MH = Marshall Islands
+country.MR = Mauritania
+country.MU = Mauritius
+country.MX = Mexico
+country.FM = Micronesia
+country.MD = Moldova
+country.MC = Monaco
+country.MN = Mongolia
+country.ME = Montenegro
+country.MA = Morocco
+country.MZ = Mozambique
+country.MM = Myanmar (Burma)
+country.NA = Namibia
+country.NR = Nauru
+country.NP = Nepal
+country.NL = Netherlands
+country.NZ = New Zealand
+country.NI = Nicaragua
+country.NE = Niger
+country.NG = Nigeria
+country.KP = North Korea
+country.MK = North Macedonia
+country.NO = Norway
+country.OM = Oman
+country.PK = Pakistan
+country.PW = Palau
+country.PA = Panama
+country.PG = Papua New Guinea
+country.PY = Paraguay
+country.PE = Peru
+country.PH = Philippines
+country.PL = Poland
+country.PT = Portugal
+country.QA = Qatar
+country.RO = Romania
+country.RU = Russia
+country.RW = Rwanda
+country.WS = Samoa
+country.SM = San Marino
+country.ST = Sao Tome and Principe
+country.SA = Saudi Arabia
+country.SN = Senegal
+country.RS = Serbia
+country.SC = Seychelles
+country.SL = Sierra Leone
+country.SG = Singapore
+country.SK = Slovakia
+country.SI = Slovenia
+country.SB = Solomon Islands
+country.SO = Somalia
+country.ZA = South Africa
+country.KR = South Korea
+country.SS = South Sudan
+country.ES = Spain
+country.LK = Sri Lanka
+country.KN = St Kitts and Nevis
+country.LC = St Lucia
+country.VC = St Vincent
+country.SD = Sudan
+country.SR = Suriname
+country.SE = Sweden
+country.CH = Switzerland
+country.SY = Syria
+country.TJ = Tajikistan
+country.TZ = Tanzania
+country.TH = Thailand
+country.BS = The Bahamas
+country.GM = The Gambia
+country.TG = Togo
+country.TO = Tonga
+country.TT = Trinidad and Tobago
+country.TN = Tunisia
+country.TR = Turkey
+country.TM = Turkmenistan
+country.TV = Tuvalu
+country.UG = Uganda
+country.UA = Ukraine
+country.AE = United Arab Emirates
+country.GB = United Kingdom
+country.US = United States
+country.UY = Uruguay
+country.UZ = Uzbekistan
+country.VU = Vanuatu
+country.VA = Vatican City
+country.VE = Venezuela
+country.VN = Vietnam
+country.YE = Yemen
+country.ZM = Zambia
+country.ZW = Zimbabwe
 
```

Comment:
I thought we agreed with Nick not to put country names in the message file. Please check and let me know as it affects my screen too.

Reply:
removed from the messages file